skimage/morphology/tests/test_util.py

```diff
@@ -0,0 +1,126 @@
+"""Test for the `_util`."""
+
+
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from skimage.morphology import _util
+
+
+class TestOffsetsToRaveledNeighbors:
```

Comment:
Why is this a class???

Reply:
To be honest it's just the way got introduced using Pytest. One might argue that is allows more descriptive test names and grouping. Prepending "offsets_to_raveled_neighbors" to every test would have resulted in quite long names in this case. I'm happy to change it though.

Reply:
To me it’s quite un-skimage-y... =P grouping can be achieved by python modules. `test_util` -> `test_offsets_to_neighbors`

Reply:
Not sure how to feel about that. I always liked that the test structure reflected to the implementation structure. That way I immediately knew where to look for the corresponding function/tests. Is there already a precedent for your suggestion?

Reply:
There’s definitely not a consistent 1-1 mapping currently in the library, it’s just gotten close to that because it’s the easiest way to get started. There’s precedent here at least: https://github.com/scikit-image/scikit-image/tree/master/skimage/graph/tests

Whether it’s the right thing is another matter, but it doesn’t seem like a big deal to me. You should still be able to find modules by function name. And indeed nothing is stopping you from making a nested folder!